test/general/tizen_device_test.dart

```diff
@@ -207,6 +280,61 @@ void main() {
     expect(tvDevice.deviceProfile, equals('tv'));
     expect(tvDevice.isSupported(), isFalse);
   });
+
+  testWithoutContext(
+      'TizenDevicePortForwarder.forwardedPorts can list forwarded ports', () {
+    final TizenDevicePortForwarder forwarder = TizenDevicePortForwarder(
+      device: _createTizenDevice(
+        processManager: processManager,
+        fileSystem: fileSystem,
+      ),
+      logger: logger,
+    );
+    processManager.addCommands(<FakeCommand>[
```

Comment:
If you use `addCommand` instead of `addCommand`s, this part can be shorter.

Reply:
Oh, you're right. I just forgot.